README.md

```diff
@@ -522,212 +675,245 @@ pre-installed on their device, follow these steps.
+With introduction of iOS 9, Apple has changed the way how deep linking should be handled in the app. Depending on which 
+scenario you want to use for your app (or if you want to use them both to support wide range of devices), you need to set 
+up your app to handle one of these or both scenarios.
+
+### <a id="deeplinking-setup-old"> Deep linking on iOS 8 and earlier
+
+Deep linking on iOS 8 and earlier devices is being done with usage of custom URL scheme setting. You need to pick some 
+custom URL scheme name which your app will be in charge for opening. This scheme name will also be used in the adjust 
+tracker URL as part of the `deep_link` parameter. In order to set this in your app, open your `Info.plist` file and add new
+`URL types` row to it. In there, as `URL identifier` write you app's bundle ID and under `URL schemes` add scheme name(s) 
+which you want your app to handle. In the example below, we have chosen that our app should handle `adjustExample` scheme 
+name.
+
+![][custom-url-scheme]
+
+After this has been set up, your app will get opened after you click the adjust tracker URL with `deep_link` parameter 
+which contains scheme name which you have chosen. After app get's opened, `openURL` method of your `AppDelegate` class will
+be triggered and the place where the content of the `deep_link` parameter from the tracker URL will be delivered. If you 
+want to access the content of the deep link, override this method.
 
 ```objc
 - (BOOL)application:(UIApplication *)application openURL:(NSURL *)url
   sourceApplication:(NSString *)sourceApplication annotation:(id)annotation {
-    [Adjust appWillOpenUrl:url];
+    // url object contains your deep link content
     
-    // Your code goes here
-    BOOL canHandle = [self someLogic:url];
-    return canHandle;
+    // Apply your logic to determine the return value of this method
+    return YES;
+    // or
+    // return NO;
 }
 ```
 
-**Important**: Tracker URLs with `deep_link` parameter and your custom URL scheme in it are no longer supported in 
-`iOS 9 and higher` and clicks on them will not cause your app to be opened nor `openURL` method to get triggered. Apple 
-dropped  support for this way of deeplinking into the app in favour of `universal links`. However, this approach will 
-`still works for devices with iOS 8 and lower`.
-
-#### <a id="universal-links">Universal links
+With this setup, you have successfully set up deep linking handling for iOS devices with iOS 8 and earlier versions.
 
-**Note**: Universal links are supported `since iOS 9`.
+### <a id="deeplinking-setup-new"> Deep linking on iOS 9 and later
 
-If you want to support [universal links][universal-links], then follow these next steps.
+In order to set deep linking support for iOS 9 and later devices, you need to enable your app to handle Apple universal 
+links. What are universal links and how does their setup look like, you can check in [here][universal-links].
```

Comment:
To find out more about universal links and how their setup looks like, you can...